Sources/HipMobileUI/ViewModels/Views/ProfileScreenViewModel.cs

```diff
@@ -54,7 +54,11 @@ private void OnChangeAppModeTapped()
 
         async void LogoutDummy()
         {
-            var result = await Navigation.DisplayAlert("Abmeldung", "Wollen Sie sich wirklich abmelden?", "Ja", "Nein");
+            var title = Strings.ProfileScreenViewModel_Dialog_Logout_Title;
```

Comment:
You don't have to create extra variables for those strings. Just reference them directly in the method `DisplayAlert(..)` and add a line break after each `Strings.X`.

Reply:
Done